.env.example

```diff
@@ -3,3 +3,4 @@ API_CLIENT_SECRET=clientsecret
 COMMUNITY_ACCOMMODATION_API_SERVICE_NAME=approved-premises
 ENVIRONMENT=local
 APPROVED_PREMISES_ROOT_PATH=approved-premises
+MAINTENANCE_MODE=false
```

Comment:
maybe slightly nitpicky of me, but would it be possible to have a more boolean-style name? `IN_MAINTENANCE_MODE` ?

Reply:
Yeah sure thing, that's a bit clearer 👍